tests/integration/targets/aci_tenant/tasks/main.yml

```diff
@@ -39,22 +39,46 @@
     annotation: ansible_test
     owner_key: ansible_key
     owner_tag: ansible_tag
+    output_path: 'cm_add_tenant.json'
```

Comment:
You want to add a step at the top of this test file to remove all files you are going to output to. There is a risk that if this task fails somewhere that at the next execution a file would be there. It is a small possibilities between run as we use docker but ansible-test retry the same test after a failed test and if the issue was just because of connectivity to APIC it might be useful to have a clean test directory.